handbook/company/pricing-features-table.yml

```diff
@@ -591,8 +594,9 @@
   usualDepartment: IT
   productCategories: [Endpoint operations,Device management]
   pricingTableCategories: [Devices]
-  demos: https://www.youtube.com/watch?v=o1G0YZAsWzI 
+  demos: 
     - description: A large tech company used scripts to fix issues with their security and compliance agents on workstations.
+      moreInfoUrl: https://www.youtube.com/watch?v=o1G0YZAsWzI 
```

Comment:
@onasismunro just checking, is it intentional that this URL is different than the one here (same video): https://github.com/fleetdm/fleet/pull/24513/files#diff-f109cc3f7421e05eddf317b7c6cc51395977e63a25dd9eed437985060ed796edR790

Reply:
Link points to the same video, but I'll adjust for consistency. CC: @onasismunro